lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc

```diff
@@ -96,25 +103,28 @@ Now that we understand what a reciprocal is, we are ready to think about how we
 @lesson-instruction{
 - Complete @printable-exercise{discover-inverse.adoc}.
 - When you're finished, complete @printable-exercise{discover-inverse-cont.adoc}.
-- What did you observe about the multiplicative inverse and its value when doing mental computation?
 }
 
-Two main ideas emerged during the previous exploration:
+@QandA{What did you observe about the multiplicative inverse and its value when doing mental computation?}
+
+@slidebreak
+
+Hopefully, you discovered two big ideas:
 
 - Dividing by @math{x} produces the same result as multiplying by its reciprocal, @math{\frac{1}{x}}.
 - Dividing by @math{\frac{1}{x}} produces the same result as multiplying by its reciprocal, @math{x}.
 
-In other words, when students see multiplication or division by a unit fraction (a fraction with a numerator of @math{1}), there is likely a path forward using mental computation only.
+In other words, when you see multiplication or division by a unit fraction (a fraction with a numerator of @math{1}), there is likely a path forward using *mental computation only*.
```

Comment:
SOMEWHERE in here I was expecting to see the word "equivalent" or "equivalence". That's the overriding theme of the course, and it definitely applies here. Is there a way to work that word in, before students see it in the following section?